test/action_controller/json/include_test.rb

```diff
@@ -36,6 +36,11 @@ def render_without_include
             render json: @author, adapter: :json
           end
 
+          def render_without_associations
+            setup_data
+            render json: @author, adapter: :json, include: []
```

Comment:
do we have a file that tests include output, specifically? or is it kinda scattered?